### PR TITLE
Elastic generated commands - preview swagger version 2024-05-16 preview

### PR DESCRIPTION
**Pull request by jkore1911**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/_associate_traffic_filter.py [hunk 1]
```diff
@@ -22,16 +22,17 @@ class AssociateTrafficFilter(AAZCommand):
...
         ]
     }
 
+    AZ_SUPPORT_NO_WAIT = True
+
     def _handler(self, command_args):
         super()._handler(command_args)
-        self._execute_operations()
-        return None
+        return self.build_lro_poller(self._execute_operations, None)
```
Comment (kairu-ms) [inline src/elastic/azext_elastic/aaz/latest/elastic/monitor/_associate_traffic_filter.py:35]: It seems this api become long running operation. Could you verify if it's expected.

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_ip_filter.py [hunk 2]
```diff
@@ -22,16 +22,17 @@ class CreateAndAssociateIpFilter(AAZCommand):
...
         ]
     }
 
+    AZ_SUPPORT_NO_WAIT = True
+
     def _handler(self, command_args):
         super()._handler(command_args)
-        self._execute_operations()
-        return None
+        return self.build_lro_poller(self._execute_operations, None)
```
Comment (kairu-ms) [inline src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_ip_filter.py:35]: Same for this command

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_pl_filter.py [hunk 3]
```diff
@@ -22,16 +22,17 @@ class CreateAndAssociatePlFilter(AAZCommand):
...
         ]
     }
 
+    AZ_SUPPORT_NO_WAIT = True
+
     def _handler(self, command_args):
         super()._handler(command_args)
-        self._execute_operations()
-        return None
+        return self.build_lro_poller(self._execute_operations, None)
```
Comment (kairu-ms) [inline src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_pl_filter.py:35]: Same for this command

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/_associate_traffic_filter.py (hunk 1 above)
Comment (jkore1911) [reply src/elastic/azext_elastic/aaz/latest/elastic/monitor/_associate_traffic_filter.py]: Changed.

File: src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_ip_filter.py (hunk 2 above)
Comment (jkore1911) [reply src/elastic/azext_elastic/aaz/latest/elastic/monitor/_create_and_associate_ip_filter.py]: We changed it to normal operation with LRO

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
https://github.com/Azure/aaz/pull/568

**Comment by jkore1911** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**